_data/opencollective.json

```diff
@@ -769,9 +783,9 @@
     "since": "Apr 14, 2021"
   },
   {
-    "name": "Kevin Sjöberg",
+    "name": "Kevin Hovsäter",
     "last_payment": 0.0,
-    "all_time": 130.0,
+    "all_time": 155.0,
     "since": "Apr 12, 2021"
```

Comment:
This looks at least a bit suspicious. Name change? Last payment is still 0 but all_time increased?

Reply:
Tracked here: https://github.com/crystal-lang/crystal-website/issues/471. I will push merge the changes for now.